src/App.js

```diff
@@ -9,18 +9,18 @@ import NotFound from './pages/NotFound';
 import Home from './pages/Home';
 import CoffeeShow from './pages/CoffeeShow'
 import { Routes, Route } from "react-router-dom"
-
-import coffees from './mockCoffees'
+import mockCoffees from './mockCoffees'
 
 function App() {
-  console.log(coffees)
+  const [coffees, setCoffees] = useState(mockCoffees)
   return (
 <>
 <Header/>
 <Routes>
       <Route path="/" element={<Home />} />
-      <Route path="/coffeeindex" element={<CoffeeIndex />} />
-      <Route path="/coffeeshow" element={<CoffeeShow />} />
+      <Route path="/coffeeindex" element={<CoffeeIndex coffees={coffees}/>} />
+      <Route path="/coffeeindex" element={<CoffeeIndex coffees={mockCoffees}/>} />
```

Comment:
Since mockCoffees are set to state, I would recommend passing coffees from state into the component.